core/src/commonMain/kotlin/com/ustadmobile/core/controller/ContentEntryList2Presenter.kt

```diff
@@ -46,26 +46,25 @@ class ContentEntryList2Presenter(context: Any, arguments: Map<String, String>, v
 
     private var showHiddenEntries = false
 
-    enum class SortOrder(val messageId: Int) {
-        ORDER_NAME_ASC(MessageID.sort_by_name_asc),
-        ORDER_NAME_DSC(MessageID.sort_by_name_desc)
-    }
+    private var selectedClazzUid: Long = 0L
 
-    class ContentEntryListSortOption(val sortOrder: SortOrder, context: Any) : MessageIdOption(sortOrder.messageId, context)
+    override val sortOptions: List<SortOrderOption>
+        get() = SORT_OPTIONS
 
     override fun onCreate(savedState: Map<String, String>?) {
         super.onCreate(savedState)
         contentEntryListItemListener.mListMode = mListMode
         contentEntryListItemListener.presenter = this
-        view.sortOptions = SortOrder.values().toList().map { ContentEntryListSortOption(it, context) }
+        selectedSortOption = SORT_OPTIONS[0]
         contentFilter = arguments[ARG_CONTENT_FILTER].toString()
```

Comment:
@SamihMustafa  - for the next commit, let's rename this arg. It's not really a filter. It's more like a display by option.  E.g. we have display by class, by parent (e.g. parent content entry),  by downloaded, etc.